### PR TITLE
Update Android instructions

### PR DESCRIPTION
**Pull request by wmora**

### CODE REVIEW

File: pages/guide/native-android-sdk.md [hunk 1]
```diff
@@ -46,23 +46,26 @@ Sync the Gradle project then add the necessary code to [initialize Smooch in you
 
 ### Initialize Smooch in your app
 
-After following the steps above, your app is setup for working with the Smooch SDK. Before your code can invoke its functionality, you'll have to initialize the library using your app's token.
+After following the steps above, your app is setup for working with the Smooch SDK. Before your code can invoke its functionality, you'll have to initialize the library using your app's id. This id uniquely identifies your app and links it to the Smooch backend that does the heavy lifting necessary to bridge the gap between you and your users.
```
Comment (mspensieri) [inline pages/guide/native-android-sdk.md]: `ID` should be capitalized wherever it's used in this doc, see https://github.com/smooch/smooch-docs/pull/366#discussion_r137006223

Comment (wmora) [reply pages/guide/native-android-sdk.md]: oh no

File: pages/guide/native-android-sdk.md [hunk 2]
```diff
@@ -46,23 +46,26 @@ Sync the Gradle project then add the necessary code to [initialize Smooch in you
...
-This token is free and uniquely identifies your app and links it to the Smooch backend that does the heavy lifting necessary to bridge the gap between you and your users.
+You can find your ID by [logging into Smooch](https://app.smooch.io) and copying it from the settings page as shown below.
 
-You can find your token by [logging into Smooch](https://app.smooch.io) and copying it from the settings page as shown below.
+![App ID on Overview Page](/images/appid.png)
 
-![App Token on Overview Page](/images/apptoken.png)
-
-Once you've located your token, use the code below to initialize Smooch.
+Once you've located your id, use the code below to initialize Smooch.
```
Comment (mspensieri) [inline pages/guide/native-android-sdk.md]: missed one

Comment (wmora) [reply pages/guide/native-android-sdk.md]: image

**Pull request merged**